client/src/pages/LogInForm.js

```diff
@@ -30,9 +30,21 @@ function LogInForm() {
 			});
 			const data = await response.json();
 			if (response.ok) {
-				const token = data.resources.token;
+				const { token, user_type } = data.resources;
+
 				localStorage.setItem("authToken", token);
-				navigate("/admin-dashboard");
+				localStorage.setItem("user_type", user_type);
```

Comment:
Just to be consistent, let's use camelCase, here too

```suggestion
				localStorage.setItem("userType", user_type);
```